docs/concepts/nuclio-real-time-functions.ipynb

```diff
@@ -167,11 +167,11 @@
     "\n",
     "Create (or update) the API gateway</br>\n",
     "It's crucial to update the defined API Gateway entity with the entity returned from store_api_gateway method. This fills in important fields like host/path, etc.\n",
-    "```\n",
+    "```python\n",
     "my_api_gateway = project.store_api_gateway(my_api_gateway)\n",
     "```\n",
-    "store_api_gateway checks that api_gateway is ready to be invoked before returning it. You can also check if api gateway is in ready state:\n",
-    "```\n",
+    "`store_api_gateway` checks that api_gateway is ready to be invoked before returning it. You can also check if api gateway is in ready state:\n",
+    "```python\n",
```

Comment:
i'm not sure the `python` magic is working in `ipynb` files, it's mostly for `md` files.